src/sentry/static/sentry/app/views/performance/table.tsx

```diff
@@ -205,7 +210,7 @@ class Table extends React.Component<Props, State> {
 
   renderHeadCellWithMeta = (tableMeta: TableData['meta']) => {
     return (column: TableColumn<keyof TableDataRow>, index: number): React.ReactNode =>
-      this.renderHeadCell(tableMeta, column, COLUMN_TITLES[index]);
+      this.renderHeadCell(tableMeta, column, getColumnTitles(this.props)[index]);
```

Comment:
nit: You can do `const column_titles = this.props.columnTitles ?? COLUMN_TITLES;` ahead of time to avoid recomputing the column titles multiple times.

Reply:
The perf impact for calling an inlined function is close enough to 0 that a 100 calls won't matter, but I'm fine with doing it out of consistency.